settings/DevHome.Settings/Views/AboutPage.xaml

```diff
@@ -1,13 +1,17 @@
-<views:DevHomePage
+<views:ToolPage
     x:Class="DevHome.Settings.Views.AboutPage"
     xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
     xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
     xmlns:ui="using:CommunityToolkit.WinUI"
     xmlns:ctControls="using:CommunityToolkit.WinUI.Controls"
     xmlns:behaviors="using:DevHome.Common.Behaviors"
     xmlns:views="using:DevHome.Common.Views"
+    xmlns:i="using:Microsoft.Xaml.Interactivity"
     behaviors:NavigationViewHeaderBehavior.HeaderTemplate="{StaticResource BreadcrumbBarDataTemplate}"
     behaviors:NavigationViewHeaderBehavior.HeaderContext="{x:Bind ViewModel}">
+    <i:Interaction.Behaviors >
+        <behaviors:AutoFocusBehavior />
+    </i:Interaction.Behaviors>

```

Comment:
Would putting this on ToolPage get us out of putting it on every page individually? (Assuming we can do it in codebehind). Otherwise, can you add something to the doc about using ToolPage to let people know they have to do this?

Reply:
That's good feedback and will help in pages that directly inherit from the tool page. However, as discussed offline some time ago, subpages will not be able to benefit from the code in ToolPage. For example, in the setup flow, we have dynamic content where the page content changes based on the step you are in during the setup flow. That said, I am not sure how common those cases are, and adding the same logic in the ToolPage code-behind is good. Updated.